src/schema_ver.rs

```diff
@@ -1,9 +0,0 @@
-/// The version of the JSON schema that `mir-json` follows. This is intended for
-/// use by downstream tools to quickly determine if they are ingesting a MIR
-/// JSON file that is compatible with the version of the schema that they are
-/// expecting.
-///
-/// Each version of the schema is assumed to be backwards-incompatible with
-/// previous versions of the schema. As such, any time this version number is
-/// bumped, it should be treated as a major version bump.
-pub const SCHEMA_VER: u64 = 5;
```

Comment:
Why did you need to move the schema version from this `src/schema_ver.rs` file?

This is a somewhat practical concern, as the [`mir-json` CI](https://github.com/GaloisInc/mir-json/blob/bf7285d76f3798fc373372eb9aa9640524b74873/.github/workflows/docker.yml#L86) currently assumes that the schema version is located in a file with that name.

Reply:
Yes, I'd favor keeping them apart. These concerns are somewhat separate: `schema_ver.rs` exists to track solely the schema version, while `version.rs` handles version types beyond just the schema version.